core/Property/Property.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Property;
+
+use Exception;
+use Piwik\Site;
+
+/**
+ * Provides access to individual properties.
+ */
+class Property extends Site
```

Comment:
Why does property/measurable extends site (and not the other way around)? Is it only a shortcut for now (will be changed later)?